src/cli/cmds/new.ts

```diff
@@ -16,6 +16,7 @@ class Command implements yargs.CommandModule {
   public builder(args: yargs.Argv) {
     args.positional('PROJECT-TYPE', { describe: 'optional only when --from is used and there is a single project type in the external module', type: 'string' });
     args.option('synth', { type: 'boolean', default: true, desc: 'Synthesize after creating .projenrc.js' });
+    args.option('removeComments', { type: 'boolean', default: false, desc: 'Synthesize .projenrc.js without commented options' });
```

Comment:
Avoid negative switches (reverse the logic and the default):


```suggestion
    args.option('comments', { type: 'boolean', default: true, desc: 'Include commented out options in .projenrc.js (use --no-comments to disable)' });
```

This also leaves us an option to allow different styles. For example, I am thinking that in a subsequent PR we can annotate options as "featured" to reduce huge number of commented out options, so we can do something like:

```
--comments=all
--comments=featured
--no-comments
```

But let's leave this for round 2.

Reply:
Good ideas, updated